ui/src/shared/components/TableGraph.js

```diff
@@ -188,7 +201,7 @@ class TableGraph extends Component {
             rowHeight={ROW_HEIGHT}
             height={tableHeight}
             width={tableWidth}
-            fixedColumnCount={1}
+            fixedColumnCount={fixedColumnCount}
```

Comment:
We'll need to implement the situation where time is in the rows, and you don't want to fix the row that it is in.  I'm noting it down.

Reply:
we want to always show the header row (the first row) and can toggle the whether the left row is fixed or not.